app/templates/app/projects.html

```diff
@@ -3,105 +3,118 @@
 {% load i18n %}
 
 {% block content %}
+<div class="filter-class">
+  <form class="filter-form">
 
-    <div class="filter-class">
-      <form class="filter-form">
-        <label for="filter">{% trans "Filter:" %}</label>
-
-            <div class="form-group">
-                <select name="subject" class="form-control">
-                    <option value="">{% trans "Subject" %}</option>
-                    {% for subject in subjects %}
-                      <option value="{{ subject.id }}" {% if request.GET.subject == subject.id|stringformat:"s" %}selected{% endif %}>
-                        {{ subject.name }}
-                      </option>
-                    {% endfor %}
-                </select>
-            </div>
+    <div class="filter-label">
+      <label for="filter">{% trans "Filter:" %}</label>
+    </div>
 
-            <div class="form-group">
-                <select name="language" class="form-control">
-                    <option value="">{% trans "Language" %}</option>
-                    {% for language in languages %}
-                      <option value="{{ language.id }}" {% if request.GET.language == language.id|stringformat:"s" %}selected{% endif %}>
-                        {{ language.name }}
-                      </option>
-                    {% endfor %}
-                </select>
-            </div>
+    <div class="form-group">
+        <label for="subject_select">{% trans "Subject" %}</label>
+        <select name="subject" class="form-control form-control-lg" id="subject_select">
+            <option value="">{% trans " All subjects" %}</option>
```

Comment:
Please remove unnecessary space at the start of the string.

Reply:
will do